src/utils/assets.js

```diff
@@ -254,6 +254,14 @@ export const addressesInclude = (addresses: string[], addressToFind: ?string): b
   return addresses.some(item => isCaseInsensitiveMatch(item, addressToFind));
 };
 
+export const isSupportedAssetAddress = (supportedAssets: Asset[], addressToCheck: ?string) => {
+  return supportedAssets.some((asset: Asset) => addressesEqual(asset.address, addressToCheck));
+};
+
+export const isSupportedAssetSymbol = (supportedAssets: Asset[], symbolToCheck: ?string) => {
+  return supportedAssets.some((asset: Asset) => asset.symbol === symbolToCheck);
+};
```

Comment:
I know we lack them in most places, but lately we've been adding return types too

Reply:
Added return types.